src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -1239,7 +1239,7 @@
     <value>{0, plural, one {There is one conflicting file name} other {There are # conflicting file names}}</value>
   </data>
   <data name="ConflictingItemsDialogSubtitleConflictsNonConflicts" xml:space="preserve">
-    <value>{0, plural, one {, and one outgoing item} other {, and # outgoing items}}</value>
+    <value>{0, plural, one {There is one conflicting file name} other {There are # conflicting file names}}, and {1, plural, one {one outgoing item} other {# outgoing items}}</value>
```

Comment:
```suggestion
    <value>There are {0, plural, one {one conflicting file name} other {# conflicting file names}}, and {1, plural, one {one outgoing item} other {# outgoing items}}</value>
```
I'm not familiar with English, but shouldn't the verb be always `are` as there are at least one conflicting item and one outgoing item?

Reply:
I am not a native English speaker, but here is/are is used correctly There is one ... and There are more ...